emojivoto-emoji-svc/cmd/server.go

```diff
@@ -21,14 +26,27 @@ func main() {
 		log.Fatalf("GRPC_PORT (currently [%s]) environment variable must me set to run the server.", grpcPort)
 	}
 
+	oce, err := ocagent.NewExporter(
+		ocagent.WithInsecure(),
+		ocagent.WithReconnectionPeriod(5 * time.Second),
+		ocagent.WithAddress(ocagentHost),
+		ocagent.WithServiceName("emoji"))
+	if err != nil {
+		log.Fatalf("Failed to create ocagent-exporter: %v", err)
+	}
+	trace.RegisterExporter(oce)
+	trace.ApplyConfig(trace.Config{
```

Comment:
Do we have any performance concerns around always sampling? Emojivoto gets used for tests every once in awhile, so maybe it should be a config option? Or does there need to be more work to figure out the configuration pushing?

Reply:
Needs more work to figure out config pushing.

Reply:
Oh.

https://github.com/census-instrumentation/opencensus-service/blob/master/receiver/opencensusreceiver/octrace/opencensus.go#L86